apps/src/util/sort.js

```diff
@@ -0,0 +1,14 @@
+// Sorts an array of objects by name, case insensitive
+export const nameComparator = (a, b) => {
+  const nameA = a.name.toUpperCase(); // ignore upper and lowercase
+  const nameB = b.name.toUpperCase(); // ignore upper and lowercase
+  if (nameA < nameB) {
```

Comment:
I didn't know you could do this with strings!

Reply:
Yeah! For reference: https://javascript.info/comparison#string-comparison